staticcheck.conf

```diff
@@ -1,2 +1,17 @@
-# See https://staticcheck.dev/docs/configuration/options/#checks.
+# In addition to the default checks [1], we also disable the following style
```

Comment:
How about:
  In addition to the checks disabled by default [1], we also ...

Reply:
Much clearer, thanks!